### DISCUSSION TITLE
2023-09-28 Community Discussion

### DISCUSSION THREAD

**Discussion by gonuke** [svalinn/DAGMC#909] [Community Calls]
Attending:
* UW-Madison: @gonuke 
* UIUC: @aprilnovak , @meltawila
* First Light Fusion: @shimwell 
* Copenhagen Atomic: @ebknudsen 
* TÜV SÜD Nuclear Technologies: @micksalt

**Comment by gonuke** [comment]
Topic: DAGMC release (w/ PyNE)

**Reply by gonuke** [reply]
Should test against newer MOAB to keep that as up-to-date as possible.

**Reply by pshriwise** [reply]
Is there a link to a failing build for this?

**Comment by pshriwise** [comment]
Topic: Exodus to DAGMC capability

**Reply by pshriwise** [reply]
As part of DAGMC proper or an independent tool?

**Reply by pshriwise** [reply]
Separation of concerns:

DAGMC: Geometry creation capabilities (set of triangles --> surfaces & volumes)
New tool: Extract and separate elements from libMesh to create new DAGMC volumes. Links libMesh and DAGMC.

**Comment by shimwell** [comment]
Topic: DAGMC training course.

**Reply by ebknudsen** [reply]
things discussed:
- Would such a training course include open source workflows?
- How would one conveniently provide access/exercise problems using cubit?
